### PR TITLE
Feature/local storage t

### PR DESCRIPTION
**Pull request by tanya2108**
What (if any) features are you implementing?
-Add and retrieve ideas from local storage. 
What (if anything) did you refactor?
- deleted console.logs and comments
Were there any issues that arose?
-WIP
Is there anything that you need from your teammate?
-look over, refactoring
Any other comments, questions, or concerns?

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,47 +1,52 @@
...
   event.preventDefault();
-  var newCard = new Idea(titleInput.value, bodyInput.value);
-  ideas.push(newCard);
-  disableSaveButton();
-  renderCard(newCard);
+  var newIdea = new Idea(titleInput.value, bodyInput.value);
   clearInput();
+  disableSaveButton();
+  ideas.push(newIdea);
+  newIdea.saveToStorage(ideas);
```
Comment (minsungpark89) [inline main.js:19]: We can delete ideas inside the parameter

File: Idea.js [hunk 2]
```diff
@@ -3,15 +3,19 @@ class Idea {
     this.id = Date.now();
     this.title = title;
     this.body = body;
-    this.isStar = false;
+    this.isStarred = false;
   }
 
   saveToStorage() {
+    var stringifyIdea = JSON.stringify(this);
```
Comment (JRicardoT) [inline Idea.js:10]: Let's change variable name to "stringifiedIdeas"

File: index.html [hunk 3]
```diff
@@ -23,83 +23,13 @@ <h3>Filter Starred Ideas</h3>
         <label for="body-input">Body</label>
         <input class="body-input" type="text" id="body" name="body" placeholder="">
       </div>
-      <button class="save-button disabled" disabled=true id="save-button">Save</button>
```
Comment (JRicardoT) [inline index.html:26]: Good call on this. We want all of our functionality within our main.js file.

**Review by minsungpark89 — COMMENTED** [review]
Good job Tanya, we'll probably have to do another final refactor tomorrow to try and get that edge case debugged.

**Review by JRicardoT — APPROVED** [review]

**Pull request merged**